src/darker/tests/test_main.py

```diff
@@ -929,3 +937,24 @@ def test_stdout_path_resolution(git_repo, capsys):
 
     assert result == 0
     assert capsys.readouterr().out == 'print("foo")\n'
+
+
+@pytest.mark.skipif(
+    not sys.platform.startswith("win"), reason="Argument length limit on Windows"
+)
```

Comment:
Wouldn't it be good to allow this test to be run on other platforms as well? It should pass everywhere, and if it suddenly won't, it would reveal a problem.

I know in this case we're looking at such stable environments as operating systems, and it's very unlikely that this would ever fail on Linux or macOS. But in principle, shouldn't we run all tests whenever we can?

Reply:
True. It’s not a specific windows test as such, and will still work under the other systems. I’ll remove the skip in the next commit.